client/spring-zeebe/src/test/java/io/camunda/zeebe/spring/client/bean/value/factory/ReadZeebeDeploymentValueTest.java

```diff
@@ -42,6 +42,7 @@ public void shouldReadSingleClassPathResourceTest() {
     ZeebeDeploymentValue expectedDeploymentValue = ZeebeDeploymentValue.builder()
       .beanInfo(classInfo)
       .classPathResources(Collections.singletonList("/1.bpmn"))
+      .bpmnLocations(Collections.singletonList("classpath*:/bpmn/**/*.bpmn"))
```

Comment:
I would prefer to have two test cases - one testing the legacy "classPathResources" attribute - and one the new "resources" attribute. I don't think it is valid to use both at the same place - maybe we should even disallow this.